evidence/clangenb/0001-clangenb-retention-rank-1.md

```diff
@@ -0,0 +1,46 @@
+# Evidence-0001: Retention at Rank 1
+
+|                 |                                 |
+| --------------- |---------------------------------|
+| **Report Date** | Date of submission (2024/07/02) |
+| **Submitted by**| clangenb                        |
+
+
+## Member details
+
+- Matrix username: `@clang:matrix.org`
+- Polkadot address: `16YCL3UVpVWQLGW3p3Zx4k5WAEp9W1DwdDnxAbyAaPxVxnp3`
+- Current rank: `1`
+- Date of initial induction: `2024/05/07`
```

Comment:
clangenb's [initial induction](ipfs://bafybeidcdu2mcvg3dwb5atfik3oz7svsimznjfqz4tmu4gmiprauqvfxei/#/extrinsics/3467894-2) was on 27th March 2024.

Reply:
Correct. It refers to the date you officially joined the Fellowship as a member.